Tests/TUSKitTests/TUSClient/TUSClient_CustomHeadersTests.swift

```diff
@@ -44,8 +44,8 @@ final class TUSClient_CustomHeadersTests: XCTestCase {
         // Make sure client adds custom headers
         
         // Expected values
-        let key = "TUSKit"
-        let value = "TransloaditKit"
+        let key = "Authorization"
```

Comment:
I think the `Authorization` header is a better example of what the custom are doing.